python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -271,7 +271,7 @@ def fit(self, X, y, convert_dtype=False):
                   convert_dtype=convert_dtype)
         return self
 
-    def predict(self, X, output_class=True, algo='auto', threshold=0.5,
+    def predict(self, X, algo='auto', threshold=0.5,
```

Comment:
should we remove this threshold too?

Reply:
(it's not in sklearn)

Reply:
This one is fine, since `predict()` produces class predictions.